src/main/java/frc/robot/Constants.java

```diff
@@ -59,4 +65,63 @@ public static class Elevator {
         public static final Dimensionless GEAR_RATIO = Rotations.of(0).div(Rotations.of(1)); // output over input
         public static final Angle MOTOR_MAX_HEIGHT = (Angle) MAX_HEIGHT.div(ELEVATOR_EXTENSION_PER_MOTOR_ANGLE).div(GEAR_RATIO);
     }
+
+    public static final class Pivot {
+        // TODO: set all of these constants, CAN IDs, and ports
+        public static final int PIVOT_MOTOR_ID = 10;
+        public static final int ROLLER_MOTOR_ID = 11;
```

Comment:
it might be more readable if we just put all of the motor IDs in the Ports constants class